### DISCUSSION TITLE
Error RcppAnnoy.h

### DISCUSSION THREAD

**Discussion by laurajchang** [satijalab/seurat#5245] [Q&A]
I am trying to install Seurat on a remote computing cluster, but keep running into the following error regarding RcppAnnoy.h. I have checked that the command `requireNamespace("RcppAnnoy")` indeed finds the namespace, but I suspect this error is arising from the fact that RcppAnnoy is installed globally on the cluster and the installation cannot be found when I try to locally install Seurat within my directory. Is there a way to specify the path to RcppAnnoy.h when installing Seurat? 

<img width="543" alt="Screen Shot 2021-10-29 at 3 24 30 PM" src="https://user-images.githubusercontent.com/4297695/139491049-d86204f7-5516-4502-a051-b7cc2e650321.png">

**Comment by JZL** [comment]
I'm not a Seurat dev and I haven't had the exact same issue, only similar, but I've found `.libPaths` getter/setter helpful for controlling where R picks up its packages.

I also have found running R within a conda environment mostly solves these problems (conda forge has most of CRAN as possible packages, it uses the prefix `r-` so for instance https://anaconda.org/conda-forge/r-seurat ) and helps with reproducibility anyway
